tests/test_resource_api.py

```diff
@@ -222,5 +243,80 @@ def test_resource_actions(self):
         self.assertEqual(resp.status_code, 200)
 
 
+class TestResourceIdApi(BaseAPITest, AssertJSONMixin, unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.test_resource = resource_type_name()
+        cls.actions = sorted(['rt:get', 'rt:put', 'rt:update', 'rt:delete'])
+        cls.app.post(
+            f'/v1/resource/{cls.test_resource}',
+            data=json.dumps({'actions': cls.actions}),
+            headers=admin_headers)
+
+        cls.app.post(
+            f"/v1/resource/{cls.test_resource}/policy/read",
+            data=json.dumps({'policy': create_test_ResourcePolicy('read', actions=['rt:get'])}),
+            headers=admin_headers
+        )
+
+        cls.app.post(
+            f"/v1/resource/{cls.test_resource}/policy/write",
+            data=json.dumps({
+                'policy': create_test_ResourcePolicy('write',
+                                                     actions=['rt:get', 'rt:put', 'rt:update', 'rt:delete'])}),
+            headers=admin_headers
+        )
+
+    def test_get_resource_ids(self):
+        """Pages of resource ids are retrieved when using the get resource API"""
+        test_resource = resource_type_name()
```

Comment:
Above you defined `cls.test_resource` so here you can use `self.test_resource` instead of defining `test_resource` again